src/api/api-client.ts

```diff
@@ -240,14 +240,15 @@ export class ApiClient {
   }
 
   addQueryParams = function (url: string, params: any) {
-    Object.entries(params).forEach(([key, value], index) => {
+    const queryParams = new URLSearchParams();
+    Object.entries(params).forEach(([key, value]) => {
```

Comment:
maybe: 
https://bobbyhadz.com/blog/javascript-convert-object-to-query-string
?

Reply:
it transforms `undefined` to a string, so we would need to filter them out first or handle undefined strings in the API
```
{
  filter: '{"status":{"eq":"ACCEPTED"}}',
  limit: 'undefined',
  nextToken: 'undefined'
}
```

Reply:
or like this: `const queryParams = new URLSearchParams(JSON.parse(JSON.stringify(params)));` ;)